src/Microsoft.AspNetCore.Mvc.Core/ApplicationParts/CompiledViewsApplicationPart.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+
+namespace Microsoft.AspNetCore.Mvc.ApplicationParts
+{
+    public class CompiledViewsApplicationPart : ApplicationPart
```

Comment:
What's the purpose of subclassing here if just does all the default stuff?

Reply:
We use this to filter by type in the feature provider. I'm open to other design suggestions to make that work.

Reply:
It doesn't the `AssemblyPart's` default stuff.